### PR TITLE
Migrate TomlFileConfigStore to @configu/integrations

### PR DESCRIPTION
**Pull request by akshay11298**
Closes https://github.com/configu/configu/issues/523

### CODE REVIEW

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]
Thanks for your PR!

please fix the tiny comment I found, so we can merge. 🙏

File: packages/integrations/stores/toml-file/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "configu"
+  ],
+  "scripts": {
+    "build": "tsup"
+  },
+  "dependencies": {
+    "@configu/integrations": "workspace:*",
+    "@configu/sdk": "workspace:*",
+    "@iarna/toml": "^2.2.5"
+  },
```
[the remark this answers is not in the document]
Comment (akshay11298) [reply packages/integrations/stores/toml-file/package.json:29]: Thanks... I have updated it

**Review by davidmeirlevy — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
@akshay11298 merged! thanks!

**Comment by davidmeirlevy** [comment]
@RichardAkman as we discussed, i ignored the lockfile issue for this PR.
we'll fix all lock problems at the end of the day.

**Pull request merged**